tests/cli/test_registry.py

```diff
@@ -23,16 +24,35 @@ class Unrelated:
 
 
 def test_find_classes():
-    gen = _find_classes(MODULE, BaseModel)
+    gen = _find_classes(MODULE, PipelineComponent)
```

Comment:
what does `gen` mean here?

Reply:
generator

Reply:
should I rename the variable? do you have a suggestion for a more clear name?